CRM/Remoteevent/Registration.php

```diff
@@ -792,6 +792,7 @@ public static function registerAdditionalParticipants(RegistrationEvent $registr
         // Create additional participants.
         foreach ($additionalParticipantsData as &$participantData) {
             $participantData['registered_by_id'] = $registration->getParticipantID();
+            $participantData['register_date'] = $registration->getParticipantData()['participant_register_date'];
```

Comment:
Is this *always* set?

Reply:
With the current state of the branch, yes, at least for *new* additional participants (as they can't currently be updated). But I was thinking of submitted values, too:

> I'm not sure if a check for existing values or values in the submission is necessary here …

… and also, whether this should default to *now* instead, as additional participants might be added later than the primary participant (think of invitations etc.)